packages/components/document-card/src/document-card.js

```diff
@@ -14,6 +14,7 @@ customElementDefineHelper(
 				dir: { type: String, reflect: true },
 				name: { type: String, reflect: true },
 				description: { type: String, reflect: true },
+				bold: { type: Boolean, reflect: true },
```

Comment:
Same as the classname, `bold` is not a good name for this property, it should be named after state of the document card. Something clear and declarative about when it should be used